### DISCUSSION TITLE
Fabric.js : overline and linethrough not applied to shadow text when affectStroke:false

### DISCUSSION THREAD

**Discussion by niketaOptimumnbrew** [fabricjs/fabric.js#9865] [Q&A]
Hi,

I'm using fabric.js lib with version 5.3.0
using text with pattern.
**issue:** overline and linethrough not applied to shadow text when affectStroke:false
Please check below fiddle to reproduce the issue.

https://jsfiddle.net/Niketa_patel/as5evzt2/42/

[
![image](https://github.com/fabricjs/fabric.js/assets/126561906/f33704b3-77c3-44ff-b429-0df5a6e4a6e6)
](url)

**Comment by asturur** [comment]
That is disabled with object:caching false because it would draw shadow over the text too.
That is why is off

**Reply by niketaOptimumnbrew** [reply]
@asturur I want object without caching.
What's the solution for this?
